cli/index.ts

```diff
@@ -439,24 +439,11 @@ function handleError(error: Error) {
 function getObjectOrJSON(options) {
```

Comment:
NIT: Not important, nor is it needed now, but it could be nice to add typing to this function at some point

Reply:
I'd be cool with adding that! For some reason, Redoc didn't feel it necessary. So, maybe we push this to the end of the project. Good find though!